src/components/BalancesList.js

```diff
@@ -169,15 +169,14 @@ export default function BalancesList() {
   );
   const balanceListItemsMemo = useMemo(() => {
     return sortedPublicKeys.map((pk) => {
-      return React.memo((props) => {
-        return (
-          <BalanceListItem
-            key={pk.toString()}
-            publicKey={pk}
-            setUsdValue={setUsdValuesCallback}
-          />
-        );
-      });
+      const key = pk.toString();
+      const Memoized = React.memo(() => (
+        <BalanceListItem
+          publicKey={pk}
+          setUsdValue={setUsdValuesCallback}
+        />
+      ));
+      return {key, Memoized};
```

Comment:
can you explain the syntax here (just for my curiosity)

i see that u are mapping all the pks from an array... but than means you are returning an array of objects from useMemo, right?

```js
[
 {key0, Memoized0},
 {key1, Memoized1},
...
]
```
is this a strong perf optimization?

Reply:
{a, b} is syntactic sugar for {a: a, b: b}
No optimization at all, I just needed to pass the key down below in the JSX tree.
But now that I think of it I can probably rewrite this more cleanly

Reply:
Ok done, should be simpler now